File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        int len = 0;
        for (int i = s.length() - 1; i >= 0; i--) {
            if (s.charAt(i) == ' ') {
                return s.substring(i, s.length() - 1).length();
            }
        }
        return s.length();
    }
}

File: 58-length-of-last-word/length-of-last-word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        s = s.Trim();
        for (int i = s.Length - 1; i >= 0; i--) {
            if (s[i] == ' ') {
                return s.Substring(i + 1).Length;
            }
        }
        return s.Length;
    }
}
